docs/rfcs/rfc-007-subscriptions.md

```diff
@@ -276,6 +276,135 @@ UNWIND meta AS m
 RETURN collect(DISTINCT m) AS meta
 ```
 
+#### Excluding types from subscriptions
+
+##### Proposed Solution:
+
+As it is currently possible to exclude specific operations for queries and mutations, it should be possible to have some exclude capabilities for subscription operations on specified types.
+
+To achieve this, the proposed solution uses the `@exclude` directive by extending the `ExcludeOperation` options from `[CREATE, READ, UPDATE, DELETE]` to include a dedicated argument `SUBSCRIPTIONS` referring to subscriptions.
```

Comment:
mmm, I'm not sure which name is better `SUBSCRIPTIONS` or `SUBSCRIBE` for this :thinking: 
Maybe we can put both alternatives in the RFC so we can discuss them at a later point

Reply:
I would choose `SUBSCRIBE` !